src/python/bcc/__init__.py

```diff
@@ -746,6 +748,19 @@ def _sym_cache(pid):
         return BPF._sym_caches[pid]
 
     @staticmethod
+    def _demangle(name):
+        if name not in BPF._demangle_caches:
+            try:
+                pipe = subprocess.Popen(
```

Comment:
nice and useful, but can we do it on c++ side just calling a function instead of a process from python?